engines/tetraedge/te/te_font3.cpp

```diff
@@ -145,9 +160,9 @@ void TeFont3::draw(TeImage &destImage, const Common::String &str, int fontSize,
 
 	uint32 uintcol = ((uint32)col.a() << fmt.aShift) | ((uint32)(col.r()) << fmt.rShift)
 						| ((uint32)(col.g()) << fmt.gShift) | ((uint32)(col.b()) << fmt.bShift);
-	Common::String line(str);
+	Common::U32String line = str.decode(_codePage);
 	if (g_engine->getCore()->language() == "he")
```

Comment:
This check actually became redundant with the changes in this PR, but can be left in for the sake of making sure it cannot affect other languages